source/includes/fact-oplog-background.rst

```diff
@@ -4,14 +4,19 @@ to the data on the destination cluster.  When operations
 that ``mongosync`` has not applied roll off the ``oplog`` 
 on the source cluster, the sync fails and ``mongosync`` exits.
 
-During the initial sync, ``mongosync`` may apply operations at a slower
-rate due to the load imposed by copying documents concurrently.
-After ``mongosync`` completes the initial sync, it applies changes 
-faster and is more likely to maintain a position in the ``oplog``
-that is close to the real-time writes occuring on the source cluster.
+Starting in version 1.5.0, {+c2c-product-name+} enables Oplog Rollover
+Resilience (ORR) which allows ``mongosync`` to apply changes on the
+source cluster to the destination cluster during the initial sync. ORR
+increases the resilience of ``mongosync`` to oplog rollover but does not
+prevent rollover entirely. For more information on ORR, see the :ref:`Oplog
+Rollover Resilience release note <1.5.0-c2c-release-notes>`.
```

Comment:
Better not to link out to release notes. Is there a location where the Release Note information lives in the docs? If not, we should probably create one.

Reply:
I think it would be nice to add more context on what ORR is. Is there a reason why we don't typically link to release notes?